raft-tech/cfgov-refresh
# Mm/improve savings function

As a user, I want to be able to save money when have extra in a green week, so I can better manage my cash flow.


---




## Changes

- When user selects a 'green week' on the month calendar, a green box appears in the same position as the Fix-it box
the button is green colored and says Save it
     - the only data element in the box will be the week-end balance .
- When the user taps the 'Save it' button, the add/edit screen for Emergency Savings
title of Emergency Savings should be changed to just Savings.
- files changed are as follows:
     - calendar.less
     - categories.js
     - ui-store.js
     - details.js


## How to test this PR

1. Open the app and clear local cache.
2. Add a starting balance of $100.
3. Navigate to the Calendar screen.  Note that you are highlighting the week that the user entered the app.  There should be a green notification box under the calendar with the description above.
4.  Add an expense for the next week of $1000.  If you tap on that week, it should be red with an accompanying _Fix It_ box under the calendar.
5.  Tap a _Green week_ and tap a _Red week_ to make sure that the box and its contents changes appropriately.
6.  Tap the _Green week_.  Tap the _Save it_ button.  You should navigate to the _Add/Edit_ screen for _Savings_ (the title should now be only _Savings_.
7.  Add a $100 Amount and Save.
8.  Navigate to the week that you added the transaction and see that it is in the transaction list at the bottom of the calendar.

## Screenshots


<img width="150" alt="Screen Shot 2020-08-13 at 6 15 15 PM" src="https://user-images.githubusercontent.com/34319929/90192504-11fcd480-dd91-11ea-9e9e-5cf8e083cbec.png">
<img width="150" alt="Screen Shot 2020-08-13 at 6 15 22 PM" src="https://user-images.githubusercontent.com/34319929/90192505-12956b00-dd91-11ea-8b9b-25cb14b42fc1.png">
<img width="150" alt="Screen Shot 2020-08-13 at 6 15 30 PM" src="https://user-images.githubusercontent.com/34319929/90192507-12956b00-dd91-11ea-8289-e662dab10914.png">
<img width="150" alt="Screen Shot 2020-08-13 at 6 15 56 PM" src="https://user-images.githubusercontent.com/34319929/90192508-12956b00-dd91-11ea-99be-167531f91e9b.png">
